shared/menubar/remote-proxy.desktop.js

```diff
@@ -67,6 +69,8 @@ const mergeProps = stateProps => ({
   loggedIn: stateProps.loggedIn,
   username: stateProps.username,
   conversations: stateProps.conversations,
+  broken: stateProps.broken,
+  following: stateProps._following.toJS(),
```

Comment:
yeah we usually won't want to use toJS() since its recursive

Reply:
I don't follow. We can't serialize an immutable map/array, so `toJS()` makes sense here.

Reply:
we usually prefer toArray() or toObject(). toJS will return a totally mixed object which flow will leave you entirely untyped

Reply:
But if we have an array of `I.RecordOf<Types._SomeType>`, then how will those serialize?

Reply:
yes, ultimately it will have to be serialized as plain js objects but its likely better to do something like
```
.toArray().map() or something vs toJS()
```

Reply:
Understood, I'll do that then.

Reply:
Actually, for `following`, that seems like overkill...

Reply:
Yeah, `.toArray()` will just work here.